Tests/TextAlignmentConstraintTests.swift

```diff
@@ -0,0 +1,223 @@
+//
+//  TextAlignmentConstraintTests.swift
+//  BonMot
+//
+//  Created by Cameron Pulsford on 10/6/16.
+//  Copyright © 2016 Raizlabs. All rights reserved.
+//
+
+import Foundation
+
+#if os(OSX)
+    import AppKit
+#else
+    import UIKit
+#endif
+
+import XCTest
+@testable import BonMot
+
+class TextAlignmentConstraintTests: XCTestCase {
+
+    private func field(withText text: String, fontSize: CGFloat) -> BONTextField {
+        let field = BONTextField(frame: .zero)
+        field.translatesAutoresizingMaskIntoConstraints = false
+
+        #if swift(>=3.0)
+            field.font = .systemFont(ofSize: fontSize)
```

Comment:
These could break if the system font metrics change. Not sure if it's worth it now, but it might be nice to change these to a hardcoded font.